client/src/routes/admin/adminGebruikerToevoegen/GebruikerToevoegen.tsx

```diff
@@ -0,0 +1,117 @@
+import {
+	Box, Button, Card, FormGroup, TextField, Typography 
+} from '@mui/material';
+import {
+	LocalizationProvider, DatePicker 
+} from '@mui/x-date-pickers';
+import {
+	AdapterDayjs 
+} from '@mui/x-date-pickers/AdapterDayjs';
+import type {
+	Dayjs 
+} from 'dayjs';
+import dayjs from 'dayjs';
+import React, {
+	useCallback 
+} from 'react';
+
+function GebruikerToevoegen() {
```

Comment:
I think we should also specify what role the user has, use the level enum to get that.